Mention how Siemens applies some patterns (#1)
siemens/InnerSourcePatterns: patterns/2-structured/communication-tooling.md

* Europace AG
* Paypal Inc.
* Mercado Libre

> * Siemens is heavily using public communication channels for all user questions and discussions around code.siemens.com.

Reviewer: maybe rather
```suggestion
* Siemens is heavily using communication channels accessible for every employee for all user questions and discussions around code.siemens.com, Inner Source and Open Source.
```

Author: Good suggestion, applying!

> * Siemens is heavily using communication channels accessible for every employee for all user questions and discussions around code.siemens.com, Inner Source and Open Source.

Reviewer 2: Maybe we could rephrase to avoid the double `for`?

```suggestion
* Siemens makes extensive use of communication channels accessible to all employees for all user questions and discussions around code.siemens.com, Inner Source and Open Source.
```

Author: That's why I love your reviews :)
